packages/datetime/src/dateInput.tsx

```diff
@@ -423,6 +417,16 @@ export class DateInput extends AbstractPureComponent<IDateInputProps, IDateInput
         }
     };
 
+    private invokeDateChangeListener(date: Date, isUserChange: boolean) {
```

Comment:
I struggled to come up with a better name for this method. I'm open to rename suggestions.

Reply:
this name works. i was considering `handleDateChange`, but i like how this isn't named like an event handler.